packages/react-components/theme-designer/src/components/Nav/Nav.tsx

```diff
@@ -1,14 +1,11 @@
 import * as React from 'react';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import { Text, BrandVariants, FluentProvider, webDarkTheme, Theme } from '@fluentui/react-components';
+import { Text, FluentProvider, webDarkTheme } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these and use `@fluentui/react-components` instead of `@griffel/react` import

```suggestion
import { makeStyles, mergeClasses, webDarkTheme, FluentProvider, Text} from '@fluentui/react-components';
```